examples/dataset/text_summarization/README.md

```diff
@@ -64,9 +64,9 @@ text summarization:
 
 ### Metrics
 
-1. [BERT F1 (`mean(result.BERT_f1)`)](https://docs.kolena.io/metrics/bertscore)
-2. [ROUGE 1 (`mean(result.ROUGE_1)`)](https://docs.kolena.io/metrics/rouge-n/)
-3. [BLEU (`mean(result.BLEU)`)](https://docs.kolena.io/metrics/bleu/)
+1. [BERT F1 (`mean(result.BERT_f1)`)](https://docs.kolena.com/metrics/bertscore)
```

Comment:
nit: `https://docs.kolena.com/metrics/bertscore/`, to be consistent with the other two. `https://docs.kolena.com/metrics/bertscore` will redirect to `https://docs.kolena.com/metrics/bertscore/` though

Reply:
haha sharp 👁️